src/applications/model/gaming-streaming-server.h

```diff
@@ -0,0 +1,103 @@
+/* -*- Mode:C++; c-file-style:"gnu"; indent-tabs-mode:nil; -*- */
+/*
+ * Copyright (c) 2020, University of Padova, Department of Information
+ * Engineering, SIGNET Lab.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation;
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ *
+ * Authors: Salman Mohebi <s.mohebi22@gmail.com>
+ *
+ */
+
+#ifndef GAMING_STREAMING_SERVER_H
+#define GAMING_STREAMING_SERVER_H
+
+#include "ns3/application.h"
+#include "ns3/event-id.h"
+#include "ns3/ptr.h"
+#include "ns3/ipv4-address.h"
+#include "ns3/random-variable-stream.h"
+#include "ns3/gaming-traffic.h"
+#include "ns3/callback.h"
+#include <functional>
+
+namespace ns3 {
+
+class Socket;
+class Packet;
+
```

Comment:
These are the Socket and Packet classes defined in NS-3.
In all other ns3 modules, they used these classes without any description, which I'm not sure if it is standard or what.
Maybe because the details of these classes can be found in the definition of each class.